src/main/java/net/jingles/enchantments/enchants/armor/Eluding.java

```diff
@@ -0,0 +1,85 @@
+package net.jingles.enchantments.enchants.armor;
+
+import net.jingles.enchantments.Enchant;
+import net.jingles.enchantments.Enchantments;
+import net.jingles.enchantments.enchants.CustomEnchant;
+import net.md_5.bungee.api.ChatColor;
+import net.md_5.bungee.api.ChatMessageType;
+import net.md_5.bungee.api.chat.TextComponent;
+import org.bukkit.NamespacedKey;
+import org.bukkit.enchantments.Enchantment;
+import org.bukkit.enchantments.EnchantmentTarget;
+import org.bukkit.entity.Mob;
+import org.bukkit.entity.Player;
+import org.bukkit.event.Event;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.entity.EntityDamageByEntityEvent;
+import org.bukkit.inventory.Inventory;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.potion.PotionEffect;
+import org.bukkit.potion.PotionEffectType;
+
+@Enchant(name = "Eluding", key = "eluding", targetItem = EnchantmentTarget.ARMOR_FEET, cooldown = 2, levelRequirement = 20,
+        description = "Grants the player a chance to be given a 3 second (multiplied by 2 per additional level) speed boost if hit by any entity while under half health")
+
+//TODO: fix the enchantment description so it's not lame af and awkward sounding
```

Comment:
boi if you had stuff TODO then don't start a PR. Your enchantment, your decision to tweak it however sksk

Reply:
I will, however, fix your bad descriptions xD